### PR TITLE
[nnfwapi] Test : Add LogSoftmax test cases

### PR DESCRIPTION
**Pull request by wateret**
Add LogSoftmax test cases, 1 positive 1 negative.

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

File: tests/nnfw_api/src/one_op_tests/LogSoftmax.cc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  _context = std::make_unique<GenModelTestContext>(cgen.finish());
+  _context->setBackends({"cpu"});
+  _context->addTestCase(uniformTCD<float>(
+      {{0, -6, 2, 4, 3, -2, 10, 1}},
+      {{-.00247565, -6.00247, -2.12692, -.126928, -.00671534, -5.00671, -.000123374, -9.00012}}));
+
+  SUCCEED();
+}
+
+// TODO Enable this test
```
Comment (wateret) [inline tests/nnfw_api/src/one_op_tests/LogSoftmax.cc:39]: Will enable this soon..

**Review by ragmani — APPROVED** [review]
LGTM
